ssh/tailssh/tailssh_test.go

```diff
@@ -990,89 +992,6 @@ func parseEnv(out []byte) map[string]string {
 	return e
 }
 
-func TestPublicKeyFetching(t *testing.T) {
```

Comment:
Unused functionality

Reply:
Yet. We were discussing at the time whether people would want this next/additionally/optionally out of paranoia or comfort.

Ideally we'd keep it and keep this PR focused.

Or if we need to delete it, do it in its own PR after a discussion.